packages/monaco/lib/editor.ts

```diff
@@ -1,6 +1,6 @@
 import type { LanguageService } from '@volar/language-service';
 import type { editor, IDisposable, MonacoEditor, Uri } from 'monaco-types';
-import { fromPosition, fromRange, toMarkerData, toTextEdit } from 'monaco-languageserver-types';
+import * as transform from 'monaco-languageserver-types';
```

Comment:
I think it’s generally better for tree-shaking to avoid namespace imports.

Reply:
I see, just revert it in https://github.com/volarjs/volar.js/commit/c04694dad6fcf3e7465df88b8ac216555453b74d